packages/forms-web-app/src/dynamic-forms/validator/string-validator.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
+const { body } = require('express-validator');
+const BaseValidator = require('./base-validator.js');
+
+class StringValidator extends BaseValidator {
+	regex = {
+		regex: '',
+		regexMessage: ''
+	};
+	minLength = {
+		minLength: 0,
+		minLengthMessage: ''
+	};
+	maxLength = {
+		maxLength: 0,
```

Comment:
maxLength should be a default maximum value like 10000 so if we don't pass in a value for max then it defaults to the largest allowable string value

Reply:
This was not a requirement. Rather a suggestion on how to implement. I have implemetned this a better way.

Reply:
The changes are required please, we either need default values or to use the optional syntax. Whether something is better is subjective but this implementation diverges from the express validator and the existing code base, both of which are suitable implementations and will provide consistency.

Reply:
You clearly do not understand what you are seeing here. Please read the unit tests.

Reply:
If defaults are required let's just get them defined with values and messages.